pkg/operator/assets/bindata.go

```diff
@@ -1594,6 +1599,16 @@ data:
       - "failed to initialize the cluster: Cluster operator monitoring is still updating"
       installFailingReason: MonitoringOperatorStillUpdating
       installFailingMessage: Timeout waiting for the monitoring operator to become ready
+    - name: AuthenticationOperatorDegraded
+      searchRegexStrings:
+      - "Cluster operator authentication Degraded is True"
+      installFailingReason: AuthenticationOperatorDegraded
+      installFailingMessage: Timeout waiting for the authentication operator to become ready
```

Comment:
That sounds great, as long as we can assume all OperatorDegraded messages are not Hive. Otherwise I'd prefer to have each degraded or group of degradeds have separate entries so that know we know if we have to take action or not.

In other words, I'd be against grouping OperatorDegraded together if we have both ones that we can ignored and ones that need action.

@dgoodwin Can we assume that all OperatorDegraded's are not Hive related (at least until shown otherwise)?

Reply:
Feels like it's probably safe to assume it's not us at this point in time, I can't think of any examples off hand where that happened and it was something we could address.

Reply:
Let's also think about other hive users who could benefit from these. As an SRE I think it would be nice to look at the ClusterDeployment CR status and see exactly which operator is degraded.

Reply:
I added a new regex called `GeneralOperatorDegraded` to catch the general operator degraded case. I made sure to add it after the more specific `AuthenticationOperatorDegraded` regex so that the specific one is matched first.